app/models/search/work_query.rb

```diff
@@ -252,6 +261,11 @@ def sort
     sort_hash
   end
 
+  # When searching outside of filters, use relevance instead of date
+  def default_sort
```

Comment:
We should mention this behaviour change in the issue so we can check for it.

Reply:
Also I think `facet_tags?` is for the filters so the conditional should be the other way around.

Reply:
Nope, it's "if we're using the filters, default to date, otherwise, default to score." (That was the behavior of the old system.)